src/utils/tokenInfo.ts

```diff
@@ -41,3 +42,11 @@ export const checkPairInWhiteList = (chainId: ChainId | undefined, symbol1: stri
     : ''
   return { isInWhiteList, data: data || {}, canonicalUrl }
 }
+
+export const getFormattedAddress = (address?: string, fallback?: string): string => {
```

Comment:
why do you need fallback here?

Reply:
additional option for anyone use it wanna fallback by custom string. Im not using it anw